<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-0MQ7D81P" failures="1" tests="1" name=" Validate_errors_are_gone " time="13.199" errors="0" timestamp="2022-05-02T21:37:27 AEST">
  <testcase classname="reusablemethods.base.TestBase" name="@AfterClass closeBrowser" time="-1.651491447426E9">
    <failure type="org.testng.TestNGException" message="
Can inject only one of &amp;lt;ITestContext, XmlTest&amp;gt; into a @AfterClass annotated closeBrowser.
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: 
Can inject only one of <ITestContext, XmlTest> into a @AfterClass annotated closeBrowser.
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass closeBrowser -->
  <testcase classname="automation.testcases.Validate_errors_are_gone" name="validateErrors" time="0.015">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.createTest(String)&amp;quot; because &amp;quot;this.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "this.extent" is null
at automation.testcases.Validate_errors_are_gone.validateErrors(Validate_errors_are_gone.java:18)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- validateErrors -->
</testsuite> <!--  Validate_errors_are_gone  -->
